tests/drivelist.spec.js

```diff
@@ -58,7 +58,7 @@ describe('Drivelist', function() {
             `Invalid error: ${device.error}`
           );
           assert.ok(
-            Number.isFinite(device.size),
+            Number.isFinite(device.size) || device.size === null,
```

Comment:
Would it make sense to swap it round to `device.size === null || Number.isFinite(device.size)` ?

Reply:
@lurch I mean, it would still behave the same and they're just tests, but it looks cleaner so I'ma change it in a sec (and a separate PR for tests only)